src/pages/favorites/favorites.component.spec.ts

```diff
@@ -71,4 +71,37 @@ describe('Favorites Component', () => {
 
     });
   });
+  describe('removeRoute', () => {
+    // Make an arbitrary route object available to each test
+    beforeAll(() => {
+      this.route = {
+        RouteId: 20035,
+        GoogleDescription: 'Campus Shuttle Southbound',
+        ShortName: '35',
+        RouteAbbreviation: '35',
+        Color: '00467E'
```

Comment:
I'll look into factories... after this. Seems like a different issue all together. I can add it.